### ISSUE TITLE
Tests failed on ECL, Clisp and ABCL

### ISSUE THREAD

**Issue by catap** [slburson/fset#42]
ECL fails as:
```
The function FSET::MAKE-CHAR is undefined..
```
and CLisp was a bit different:
```
:info:test *** - Test failed:
:info:test        (NEW-LET:LET
:info:test         ((FSET::HT
:info:test           (FSET:CONVERT 'HASH-TABLE (FSET:MAP (1 2) (3 4)) :TEST 'EQUAL)))
:info:test         (EQL (HASH-TABLE-TEST FSET::HT) 'EQUAL))
```
and ABCL is also different:
```
:info:test ; Compilation unit finished
:info:test ;   Caught 1 WARNING condition
:info:test Caught UNBOUND-VARIABLE while processing --eval option "(asdf:operate (quote asdf:test-op) (quote fset))":
:info:test   The variable LIST-17623 is unbound.
```

**Comment by hraban** [comment]
Confirmed for clisp, fwiiw.

**Comment by slburson** [comment]
Fixed and tested on ABCL 1.9.2.

I wrote `make-char` for ECL, but the test suite does not complete for me on any of the last three ECL versions; I get a recursive-lock error which is probably spurious.  I also encountered a pretty nasty-looking [ECL bug](https://gitlab.com/embeddable-common-lisp/ecl/-/issues/745), so I'll wait until they fix that and try again.

The most recent CLisp release is from 2010, and didn't build for me when I tried it just now (using gcc 7.5.0 on Linux).  I conclude that CLisp is dead.

Considering this issue fixed until further notice.
